### ISSUE TITLE
Supplementary Report - Final Review Page

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-reporting#480]
#### Description:
The Final Review page of an annual report in supplementary report mode has an alert and an additional field
 [Final Review page Wireframe](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=20190-4582&t=XJfevJ7Ha4Eekqpn-4)

#### Acceptance Criteria:

~~Given I am on the Final review page in supplementary report mode
Then I see an alert 
And I see that there is a new section called Reason for Change and a field from the Review Changes page called Reason for Change (dependent on #479)~~

Given I am on the Final Review page in Supplementary Report mode
Then the new "Reasons for Errors" page in visible on this page. See #729 

*Question for devs: could we put the Reasons for Error page at the top of the final review page? If it's easier to lay out each form in order, then we can keep it at the bottom. But as a UX improvement it would be helpful to show them the reasons for error info right at the top of this form. 

<img width="658" alt="Image" src="https://github.com/user-attachments/assets/c75f68d3-3117-4a8a-8974-6bba108123d4" />

Given I am on the final review page,
Then I see the changed values only and not the original values.

#### Development Checklist:

- [ ]
- [ ] Use GET endpoint for is_supplementary_report_version endpoint as a flag to conditionally render supplement report fields in the schemas  (see `@reporting/src/app/utils/getIsSupplementaryReport`)
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

#### Definition of Ready (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

#### Definition of Done (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By/Blocking
#729 

#### Notes:

-

**Comment by patriciarussellCAS** [comment]
@zoeyli-46 I've updated this ticket based on the changes we're making to the Reason for Error (#729)/Review changes (#479) tickets. Let me know if this works for you. 
I'm still not sure about the banner component, so left it out, but can add that back in if we think it's necessary. Thanks!

**Comment by zoeyli-46** [comment]
Removed banner component from final review, because it will be redundant in addition to seeing it on the review changes/reason for edit page

**Comment by shon-button** [comment]
@patriciarussellCAS @zoeyli-46 

Should "Reasons for Errors" page in visible on `Submitted` page? 

Thanks very much.

**Comment by zoeyli-46** [comment]
hey @shon-button thanks! just to confirm, do you mean whether reasons for errors fields should be visible on this submitted page? 

<img width="707" alt="Image" src="https://github.com/user-attachments/assets/2c340167-4bfd-4695-8fc0-be7f5d484b77" />

**Comment by shon-button** [comment]
> hey [@shon-button](https://github.com/shon-button) thanks! just to confirm, do you mean whether reasons for errors fields should be visible on this submitted page?
> 
> <img alt="Image" width="707" src="https://private-user-images.githubusercontent.com/77090142/444280151-2c340167-4bfd-4695-8fc0-be7f5d484b77.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDczNDM1NjUsIm5iZiI6MTc0NzM0MzI2NSwicGF0aCI6Ii83NzA5MDE0Mi80NDQyODAxNTEtMmMzNDAxNjctNGJmZC00Njk1LThmYzAtYmU3ZjVkNDg0Yjc3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA1MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNTE1VDIxMDc0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc3NmYzMGVhZDM5YWUwODVmMmNmYTFmYWI5YjJhZTNlOGRlYjA2OGIyYzhiYWNiYjlhNzAxZTk5ODYwZjc0OGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.SJJd6j0HE376rOgdNmdk82SfFIb6e_xgZEq88VN0LaE">

@zoeyli-46 
I mean here: 
thanks.
![Image](https://github.com/user-attachments/assets/42c0cd43-a62e-49e3-bad7-599a77ab98d5)

**Comment by patriciarussellCAS** [comment]
@shon-button - it should look the same as the final review page, so yes, we'd want to see it on the "view only" version of the submitted report. Thanks!

**Comment by zoeyli-46** [comment]
> [@zoeyli-46](https://github.com/zoeyli-46) I mean here: thanks. 
Hey @shon-button yes, reason for edits should appear on that page at the top, just as you have it in the screenshot. thanks!

**Comment by fviduya** [comment]
I have tested this locally alongside #729, moving to Pending PO for review. Thank you!

**Comment by patriciarussellCAS** [comment]
Not working in DEV.

**Comment by zoeyli-46** [comment]
Created a separate ticket for a bug on this page: [793](https://github.com/orgs/bcgov/projects/123/views/1?pane=issue&itemId=112819519&issue=bcgov%7Ccas-reporting%7C793) when a supplementary report has been created,  clicking back on the final review page takes them to the Submitted page view, and clicking back just toggles them between the 2 pages and it’s not possible to go back anymore, this happens only when there are excess emissions. Here’s a loom [recording](https://www.loom.com/share/74ddd2647b0b4116bd1f52913c669abf)

**Comment by patriciarussellCAS** [comment]
Working in DEV now (other than the back button as noted by Zoey) - moving to accepted.

**Comment by fviduya** [comment]
Working in TEST. The Reason for Edits is visible for a supplementary report and is required.
